commands/project_sync.go

```diff
@@ -119,20 +119,22 @@ func (cmd *ProjectSync) RunStart(ctx *cli.Context) error {
 
 // StartUnisonSync will create and launch the volumes and containers on systems that need/support Unison
 func (cmd *ProjectSync) StartUnisonSync(ctx *cli.Context, volumeName string, config *ProjectConfig, workingDir string) error {
+	cmd.out.Spin("Starting Outrigger Filesync (unison)...")
+
 	// Ensure the processes can handle a large number of watches
 	if err := cmd.machine.SetSysctl("fs.inotify.max_user_watches", maxWatches); err != nil {
 		cmd.Failure(fmt.Sprintf("Failure configuring file watches on Docker Machine: %v", err), "INOTIFY-WATCH-FAILURE", 12)
 	}
 
-	cmd.out.Channel.Info.Printf("Starting sync volume: %s", volumeName)
+	cmd.out.Info("Starting sync volume: %s", volumeName)
 	if err := util.Command("docker", "volume", "create", volumeName).Run(); err != nil {
 		return cmd.Failure(fmt.Sprintf("Failed to create sync volume: %s", volumeName), "VOLUME-CREATE-FAILED", 13)
 	}
-
-	cmd.out.Info("Starting Unison container")
+	cmd.out.Info("Sync volume '%s' created", volumeName)
```

Comment:
I never see this message, even recording and playing in slow motion.

Reply:
On further testing, I don't see it without verbose mode but I do see it in verbose mode.